build-aux/m4/bitcoin_qt.m4

```diff
@@ -423,14 +424,24 @@ AC_DEFUN([_BITCOIN_QT_FIND_LIBS_WITH_PKGCONFIG],[
         PKG_CHECK_MODULES([QT_DBUS], [${QT_LIB_PREFIX}DBus], [QT_DBUS_INCLUDES="$QT_DBUS_CFLAGS"; have_qt_dbus=yes], [have_qt_dbus=no])
       fi
     ])
+    BITCOIN_QT_CHECK([
+      AC_CACHE_CHECK(for > Qt 5.4,bitcoin_cv_qt55,
+        PKG_CHECK_EXISTS(Qt5Core < 5.5.1,[bitcoin_cv_qt55=no],[bitcoin_cv_qt55=yes])
+      )
+    ])
+    BITCOIN_QT_CHECK([
+      AC_CACHE_CHECK(for > Qt 5.7,bitcoin_cv_qt58,
+        PKG_CHECK_EXISTS(Qt5Core < 5.8.0,[bitcoin_cv_qt58=no],[bitcoin_cv_qt58=yes])
```

Comment:
Message should mention `> Qt 5.8` to be consistent

Reply:
I think it is consistent. "Check that QT > 5.7" means that it is 5.8 or more. This is the same as above: "check that QT > 5.4", that means it is 5.5 or more. The same message is used some rows above (275):
```
AC_CACHE_CHECK(for > Qt 5.7, bitcoin_cv_qt58,[
AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
```

Reply:
I think that's confusing. `> Qt 5.7` means newer than 5.7, so may also mean 5.7.1 or whatever, at least at a cursory glance.

If you're concerned about lawyering correctness of `>` versus `>=` feel free to make it `>=5.8` instead.

Reply:
I think you are right. 5.7.1 is a good example. I'll amend the commit.